core/src/main/scala/caliban/GraphQL.scala

```diff
@@ -30,6 +30,7 @@ trait GraphQL[-R] { self =>
   protected val additionalDirectives: List[__Directive]
   protected val features: Set[Feature]
   protected val transformer: Transformer[R]
+  protected val additionalValidations: List[Validator.QueryValidation]
```

Comment:
I had a quick look through the PR and couldn't find any usages of this. What is it being used for?